backend/src/main/java/wooteco/retrospective/config/LoginInterceptor.java

```diff
@@ -0,0 +1,40 @@
+package wooteco.retrospective.config;
+
+import org.springframework.http.HttpMethod;
+import org.springframework.web.servlet.HandlerInterceptor;
+import wooteco.retrospective.exception.AuthorizationException;
+import wooteco.retrospective.infrastructure.auth.AuthorizationExtractor;
+import wooteco.retrospective.infrastructure.auth.JwtTokenProvider;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LoginInterceptor implements HandlerInterceptor {
+
+    private JwtTokenProvider jwtTokenProvider;
+
+    public LoginInterceptor(JwtTokenProvider jwtTokenProvider) {
+        this.jwtTokenProvider = jwtTokenProvider;
+    }
+
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) {
+
+        if (request.getMethod().equals(HttpMethod.OPTIONS.name())) {
```

Comment:
```suggestion
        if (HttpMethod.OPTIONS.name().equals(request.getMethod())) {
```

request.getMethod()를 했을 때 null이 나오는 경우가 있을까? 생각이 들었지만,
이 순서로 작성하면 NPE를 방지할 수 있을 거 같아요!